CSS/Components/button.css

```diff
@@ -0,0 +1,98 @@
+.btn{
+    padding: 0.4rem 0.6rem;
+    border-radius: 5px;
+    font-size: 1.2rem;
+    font-weight: 500;
+    margin: 0 10px;
+    border: none;
+    background-color: var(--primary-color);
+    color: var(--off-white);
+    cursor: pointer;
+}
+.btn:focus {
+    outline: none;
+  }
+  /* Basic Buttons */
+.btn-primary {
+    background-color: var(--primary-color);
+  }
+  
+  .btn-secondary {
+    background-color: var(--secondary-color);
+  }
+  .btn-success {
+    background-color: var(--success-color);
+  }
+  .btn-danger {
+    background-color: var(--danger-color);
+  }
+  .btn-warning {
+    background-color: var(--warning-color);
+  }
```

Comment:
These classes are only changing the background colo so you instead of naming them like `.btn-primary, .btn-secondary etc.`, you can name them like `.bg-primary | secondary | success etc.` so that its more descriptive and can be used with other components as well

Reply:
Makes sense skip , thanks !